src/ai/.x/templates/openai-chat-w-asst-webpage-js/src/OpenAIAssistantsStreamingClass.js

```diff
@@ -0,0 +1,99 @@
+<#@ template hostspecific="true" #>
+<#@ output extension=".js" encoding="utf-8" #>
+<#@ parameter type="System.String" name="ClassName" #>
+const { OpenAI } = require('openai');
+
+class <#= ClassName #> {
+
+  // Create the class using the Azure OpenAI API, which requires a different setup, baseURL, api-key headers, and query parameters
+  static createUsingAzure(azureOpenAIAPIVersion, azureOpenAIEndpoint, azureOpenAIKey, azureOpenAIDeploymentName, openAIAssistantId) {
+    console.log("Using Azure OpenAI API...");
+    return new <#= ClassName #>(openAIAssistantId,
+      new OpenAI({
+        apiKey: azureOpenAIKey,
```

Comment:
We should add a disclaimer asking users not to deploy their OpenAI API key on client side, as described [here](https://help.openai.com/en/articles/5112595-best-practices-for-api-key-safety).

Reply:
done.

Reply:
done.